bootconfig/README.md

```diff
@@ -0,0 +1,53 @@
+# Bootconfig API
+
+The bootconfig service is responsible for the setting of bootstrap configuration for the device after initial Bootz bootstrap.
+This message may include both the base device configuration but also the new gNSI artifacts.
+
+While this is service which can be used out of band it also allows for the development of an industry standard CLI for recovery
+of devices.
+
+## RPC use case
+
+* From remote rpc client call `gnoi.Bootconfig.Set` using bootconfig message containing elements to override.
+
+## CLI use cases
+
+The CLI should be bundled with the router and exist in the default path.
+The CLI must be named `oc_cli` it must have the following heirarchy for calling specific services.
+
+```
+oc_cli
```

Comment:
Given that we don't have test cases for the non-gNOI cases here, are they as critical?

Reply:
they are probably not p0 as if we can recover. the box with the bootconfig then we can use other tools for troubleshooting after recovery